grid/recruitment/create-candidate/maps/recordings/breezy-hr.recording.json

```diff
@@ -255,16 +280,237 @@
           "X-RateLimit-Limit",
           "100",
           "X-RateLimit-Remaining",
-          "95",
+          "96",
           "X-RateLimit-Reset",
-          "1670421590",
+          "1672231691",
           "ETag",
           "W/\"61-VOmYEQW8LNhNin8YPuANSEMH9y8\"",
           "Vary",
           "Accept-Encoding"
         ],
         "responseIsBinary": false
       }
+    ],
+    "0de0fe70126c707d7921a8307e36c0ec": [],
+    "29a5bf03ceca5dbd3e7d244d42b01174": [],
+    "4d58e4b9dbfa220a6ddad83fe9d25e7e": [],
+    "8d1596557a6ba534defec4d4d9016532": [],
+    "cfa81861f9e39455b6c83fc15d1942da": [],
+    "f4dc24e7230ff97506452f9a9ecaa2c2": [],
+    "1f557bdc0faa8ff1c45d6772a897d773": [
+      {
```

Comment:
Aren't these duplicated? (see also bellow)

Reply:
Fixed, but I am not sure what is the right testing flow to avoid duplications. Looks like I have to delete file with recording before calling 
```
yarn test:record grid/recruitment/create-candidate/maps/breezy-hr -u
```
in order to avoid duplicate snapshots.

Reply:
Currently the best way is to always try the test with mocked traffic and either:
- fix a hash for an old recording to one that can't be found (Mentioned in error message)
- or remove recordings that you think are duplicated and rerecord